### ISSUE TITLE
Github Milestones - best practices

### ISSUE THREAD

**Issue by Jbarget** [dwyl/process-handbook#37]
https://github.com/dwyl/tudo/issues/36

Moving issue above into this repo as seems to be more relevant:

**Comment by Cleop** [comment]
The original issue description was: 

> Research into best practises on how milestones are used on Github and decisions made on how our app uses them and links them with labels.

@iteles - I don't know if you think I'm over-simplifying things by saying - we use milestones to outline sprints. They are use to group issues into dated periods of time so that we can work with our client to manage and discuss what we believe we can achieve and what is most important to achieve in that time. Do we want this recorded somewhere in a readme? Or is this better off in the product-owner repo as this is probably a useful explanation for a product owner? or it could be mentioned in the sprint planning bit in the readme of this repo?

**Comment by iteles** [comment]
@Cleop Agreed, we've been using them in that way and it's worked well for us (the original issue was from July 2015 !). Let's record it in the PO repo but put in a reference to it from here.
